k8s-deployer/scripts/publish-report.sh

```diff
@@ -63,13 +71,17 @@ CONTENT_DIR_PATH="../${CONTENT_DIR}"
 pwd
 
 git clone $GIT_REPO . && \
+  cp .git/config .git/config_backup && \
+  git config user.email "${USERNAME}@kindredgroup.com" && \
```

Comment:
Same here for `@kindredgroup.com` email suffix

Reply:
Yes, created ticket already.